src/app/operators/components/operator-examples/operator-examples.component.html

```diff
@@ -1,11 +1,11 @@
-<h2> Examples </h2>
+<h2> {{ 'OPERATOR.EXAMPLES' | translate }} </h2>
 <div
   class="code-example"
   *ngFor="let example of operatorExamples"
   appHighlightJs>
   <div class="code-block mat-elevation-z2">
     <div class="example-header">
-      <div class="header-title" [innerHTML]="example.name"></div>
+      <div class="header-title" [innerHTML]="example.name[currentLang]"></div>
```

Comment:
I think it would make sense to write a pipe for this behaviour. To get the element at currentLang will be pretty repetiv.